pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
So you are saying it's OK for the user's ISR to be triggered when the INT pin is inactive just because it went inactive a nanosecond after your check for the INT pin level?